src/pages/movie-page/movie-page.test.js

```diff
@@ -0,0 +1,38 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { MoviePage, movieLoader } from './movie-page';
+import { MovieService } from '../../services/movie-service';
+
+const mockMovie = {
+  id: 0,
+  title: 'the movie',
+}
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useLoaderData: jest.fn(() => ({}))
+}));
+
+describe('Movie page', () => {
+  it('Has button in the document', () => {
```

Comment:
what button? Just a random one?

Reply:
Changed the description to `'Has a close button in the document'`.